devel/libtool/Portfile

```diff
@@ -80,12 +80,9 @@ post-destroot {
 
     # libtool 2.4.6 contains com.dropbox.attributes xattrs,
     # let's avoid spreading them further (#54459)
-    # xattr on leopard or earlier does not support -r so do the recursion manually
-    if {${os.platform} eq "darwin" && ${os.major} >= 10} {
-        system "xattr -r -d com.dropbox.attributes ${destroot}"
-    } else {
-        system -W ${destroot} "find . -type f -print0 | xargs -0 xattr -d com.dropbox.attributes"
-    }
+    # xattr on Leopard or earlier does not support -r so do the recursion manually
+    # xattr on Monterey supports -r but errors on directories, so do fully manual
```

Comment:
I trust you have reported this bug to Apple? Including the feedback URL (even if non-Apple employees cannot access it) in the comment would be useful.

Reply:
I haven't yet. I'll update the OS & Xcode (if necessary) & verify, and if this behavior remains then I'll submit a feedback report to Apple.